nixos/modules/misc/mandoc.nix

```
@@ -5,59 +5,191 @@ let
 
   cfg = config.documentation.man.mandoc;
 
-in {
+  toMandocOutput = output: lib.concatLines (
+    lib.mapAttrsToList
+      (
+        name: value:
+          if lib.isString value then "output ${name} ${value}"
+          else if lib.isInt value || lib.isPath value then "output ${name} ${builtins.toString value}"
+          else if value == true then "output ${name}"
+          else ""
```

Let's make the assumption clear here to avoid introducing a weird bug later.

```suggestion
          else assert value == null; ""
```

---

Will this print confusing output like "assertion (x == null) failed"?

---

Yes, but I'd expect the module system type checker to fail first (with a helpful message), so this should only fire if there's a bug in the module. Maybe you can try when making the change.